ansible/provision-and-execute-tests/roles/install_fluent_bit_from_gh_prerelease/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Check mandatory variables
+  assert:
+    that:
+      - gh_prerelease_tag is defined
+      - fb_package_name is defined
+
+- name: Install (replace) Fluent Bit package
+  include_tasks: "{{ ansible_pkg_mgr }}.yml"
```

Comment:
can't you `become` here and avoid repeating it in each yaml file?

Reply:
Good point! Nevertheless, it seems the become keyword is [not supported in include_tasks](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_tasks_module.html):

![Screenshot 2023-10-04 at 11 54 44](https://github.com/newrelic/fluent-bit-package/assets/6243832/e7c52167-8b82-4f06-a8dc-c1ae622f04ab)